### ISSUE TITLE
Selfhosted Gitlab not detected in PRs

### ISSUE THREAD

**Issue by bakoff44** [devlint/GitWand#168]
Hi,

I'm currently using GitWand 3.7.2.
My Gitlab account is configured with `glab`, and it works great.

But in the PRs interface, Github is still kicking, instead of Gitlab.

-----

Glab auth status : 
``` shell
glab auth status
forge
  ✓ Logged in to forge as jpac*** (keyring)
  ✓ Git operations for forge configured to use ssh protocol.
  ✓ API calls for forge are made over https protocol.
  ✓ REST API Endpoint: https://forge/api/v4/
  ✓ GraphQL Endpoint: https://forge/api/graphql/
  ✓ Token found in operating system keyring: **************************
``` 

-----

GitWand account : 
<img width="736" height="133" alt="Image" src="https://github.com/user-attachments/assets/151d2cd0-01b8-44c5-a244-efd9a2154eeb" />

-----
GitWand PRs
<img width="804" height="641" alt="Image" src="https://github.com/user-attachments/assets/6b031ebd-daa6-4d6d-a31e-bb5047f2e2ea" />

**Comment by devlint** [comment]
Thanks for the detailed report, this is now fixed in v3.7.3.

Root cause: GitWand's forge detection only recognized a GitLab remote by matching the literal substring `"gitlab"` in the URL. Your self-hosted instance's hostname (`forge`) doesn't contain that word, so detection fell through to `"unknown"` and the PR panel silently defaulted to GitHub, even though `glab` was correctly authenticated.

The fix: when the URL doesn't give the forge away, GitWand now asks the CLI itself, checking whether `glab` or `gh` is authenticated for that exact host (`auth status --hostname <host>`), the same way those tools already resolve it internally. Whichever one confirms the host wins, so a self-hosted GitLab instance is now recognized regardless of what its hostname looks like, as long as you've already logged into it via `glab auth login`.

v3.7.3 is up now (https://github.com/devlint/GitWand/releases/tag/v3.7.3) — the auto-updater should pick it up, or you can grab it directly from the releases page. Let us know if it's still not detecting correctly after updating.
